tests/output/dbus/DbusDocumentTest.py

```diff
@@ -83,5 +83,5 @@ def test_analyze(self):
         uut.SetConfigFile(self.config_path + "2")
         output = uut.Analyze()
         self.assertEqual(output[0], 255)
-        self.assertEqual(output[1][1]["log_level"], "ERROR")
-        self.assertEqual(output[1][1]["message"], Constants.CRASH_MESSAGE)
+        self.assertEqual(output[1][0]["log_level"], "ERROR")
+        self.assertEqual(output[1][0]["message"], Constants.CRASH_MESSAGE)
```

Comment:
Oh, I see now, when I use `coala-ci -b=SpaceConsistencyBear' it's printed an error.